backend/src/main/java/codezap/global/auditing/SkipModifiedAtUpdateBaseTimeEntity.java

```diff
@@ -0,0 +1,43 @@
+package codezap.global.auditing;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.MappedSuperclass;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.PostPersist;
+import jakarta.persistence.PostUpdate;
+import jakarta.persistence.PreUpdate;
+import jakarta.persistence.Transient;
+
+import lombok.Getter;
+
+@Getter
+@MappedSuperclass
```

Comment:
컨벤션에 맞게 순서 변경도 제안합니당

```suggestion
@MappedSuperclass
@Getter
```

Reply:
해당 클래스에 Getter 가 필요하지 않을 것이라 판단하여 @Getter를 제거하는 방안으로 해결했습니다~